README.md

```diff
@@ -254,22 +298,49 @@ Rspec.desrcibe TShirt do
 end
 ```
 
-This is the best test file yet. Because we are not using `before` nor `let`, we don't need to search around our test examples to figure out what values are in use. We have reduced the cognitive burn of understanding our test code, and thus made them easier to understand and edit/improve as our `TShirt` class evolves. We also quickly understand what are tests do NOT need.
+This is the best test file yet. Because we are not using `before` nor `let`, we don't need to search around our test examples to figure out what values are in use. We have reduced the cognitive burn of understanding our test code, and thus made them easier to understand and edit/improve as our `TShirt` class evolves. We also quickly understand what are tests do NOT need. `before` and `let` can easily lead to the testing [Obscure Test](http://xunitpatterns.com/Obscure%20Test.html), [Fragile Test](http://xunitpatterns.com/Fragile%20Test.html#Fragile%20Fixture), and Slow Tests](http://xunitpatterns.com/Slow%20Tests.html) testing smells. Avoiding `before` and `let` can help us avoid these smells.
+
+Nitro tests suffer greatly from issues of over using `before` and `let` helper methods. As do some learn.co labs. Understanding these method drawbacks puts you a fantastic position to help Nitro clean up its tests. To learn more about why `before` and `let` should be avoided, [read this](https://robots.thoughtbot.com/lets-not).
+
+# Example structure
+
+When we write examples, we should try to structure the example so it tells a story. Good testing stories take this form:
+
+1. Setup
+1. Exercise
+1. Verify
+1. Teardown
+
+Or:
+
+1. Setup
+1. Exercise / Verify
+1. Teardown
+
+In a language like Ruby we can usually ignore Teardown. So lets just focus on Setup, Exercise, and Teardown. This is called a [Four-Phase Test and can be read about here](https://robots.thoughtbot.com/four-phase-test).
+
+## Setup
+
+This is where we create the testing fixtures and setup the system under test.
```

Comment:
"testing fixtures" seems like it might be jargon-y for the intended audience. What do you think about something like "This is where we prepare the system to test our scenario. Typically this requires us to initialize parts of the system with a known state so we may make assertions about the behavior of the interactions we intend to test."?